interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,476 @@
+            </properties>
+          </leafNode>
+          <leafNode name="subnets">
+            <properties>
+              <help>Declare network segment</help>
+              <valueHelp>
+                <format>ipv4net</format>
+                <description>IPv4 address and prefix length</description>
+              </valueHelp>
+              <valueHelp>
+                <format>ipv6net</format>
+                <description>IPv6 address and prefix length</description>
+              </valueHelp>
+              <constraint>
+                <validator name="ipv4-prefix"/>
+                <validator name="ipv6-prefix"/>
+              </constraint>
+              <multi/>
+            </properties>
+          </leafNode>
+          <leafNode name="bind-address">
+            <properties>
+              <help>Bind To Address</help>
+              <valueHelp>
+                <format>ipv4net</format>
+                <description>IPv4 address and prefix length</description>
+              </valueHelp>
+              <valueHelp>
+                <format>ipv6net</format>
+                <description>IPv6 address and prefix length</description>
+              </valueHelp>
+              <constraint>
+                <validator name="ipv4-address"/>
+                <validator name="ipv6-address"/>
+              </constraint>
+              <multi/>
+            </properties>
+          </leafNode>
+          <leafNode name="local-address">
+            <properties>
+              <help>IP address</help>
+              <valueHelp>
+                <format>ipv4</format>
+                <description>IPv4 address</description>
+              </valueHelp>
+              <valueHelp>
+                <format>ipv6</format>
+                <description>IPv6 address</description>
+              </valueHelp>
+              <valueHelp>
+                <format>host</format>
+                <description>Host</description>
+              </valueHelp>
+              <constraint>
+                <validator name="ip-host"/>
+              </constraint>
+              <multi/>
+            </properties>
+          </leafNode>
+          <leafNode name="port">
+            <properties>
+              <help>Port Option</help>
+              <constraint>
+                <validator name="numeric" argument="--range 1-65535"/>
+              </constraint>
+            </properties>
+            <defaultValue>655</defaultValue>
+          </leafNode>
+          <leafNode name="bind-interface">
+            <properties>
+              <help>Bind To Interface</help>
+              <completionHelp>
+                <script>${vyos_completion_dir}/list_interfaces.py --broadcast</script>
+              </completionHelp>
+            </properties>
+          </leafNode>
+          <leafNode name="connect">
+            <properties>
+              <help>Connect To Peer Node Name</help>
+            </properties>
+          </leafNode>
```

Comment:
I'm thinking about how to improve this, but I don't think there is a need to specify its path, because tinc requires that the peer of `ConnectTo` must have a configuration under the Host path